.changeset/moody-numbers-brush.md

```diff
@@ -0,0 +1,22 @@
+---
+"@logto/connector-vonage-sms": major
+---
+
+add vonage sms connector
+
+You can now send SMS messages using the Vonage SMS connector.
+
+To work with this connector, you will need to [sign up for an account](https://developer.vonage.com/en/account/guides/dashboard-management#create-and-configure-a-vonage-account) in Vonage. This will give you an API key and secret that you can use to access the APIs through this connector.
+
+Once you have an account, you can find your API Key and API Secret at the top of the Vonage API Dashboard.
```

Comment:
nit:
```suggestion
Once you have an account, you can find your API key and API secret at the top of the Vonage API Dashboard.
```

Reply:
If you would like to keep `API Key` then you probably need to change the letter case at line 9